test/mountain_view/component_test.rb

```diff
@@ -16,16 +16,112 @@ def test_humanized_title
   def test_styleguide_stubs
     component = MountainView::Component.new("header")
 
-    expected_stub = {
-      id: 1,
-      title: "20 Mountains you didn't know they even existed",
-      subtitle: "Buzzfeed title"
-    }
+    expected_stub =
+      {
+        :meta => "There is this different classes",
+        :stubs =>
+          [
+            { :id  =>  1, :title => "20 Mountains you didn't know they even existed",
+             :subtitle => "Buzzfeed title"
+            },
+            { :id => 2, :title => "You won't believe what happened to this man at Aspen" }
+          ]
+      }
 
     assert_instance_of Array, component.styleguide_stubs
     assert_equal expected_stub, component.styleguide_stubs.first
   end
 
+  def test_component_stubs
+    component = MountainView::Component.new("header")
+    expected_stub =
+      [
+        { :id => 1, :title => "20 Mountains you didn't know they even existed",
+         :subtitle => "Buzzfeed title" },
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.